smaug/python/ops/data_op_test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+import unittest
+import numpy as np
+
+from smaug.core import types_pb2
+from smaug.python.tensor import Tensor
+from smaug.python.graph import Graph
+from smaug.python.ops import data_op
+from smaug.python.ops import math_ops
+
+graph_name = "test_graph"
+backend = "Reference"
+x = Tensor(
+    data_layout=types_pb2.N, tensor_data=np.random.rand(4).astype(np.float32))
+y = Tensor(
+    data_layout=types_pb2.N, tensor_data=np.random.rand(4).astype(np.float32))
+
+def get_num_data_nodes(graph):
+  return len(
```

Comment:
It's more efficient to just do a regular for loop that increments a counter, instead of creating a temporay list only to just get its length.

Reply:
Right. Changed to a regular loop to count the length.